### PR TITLE
Added personal blurb

### PR DESCRIPTION
**Pull request by nquesada**

### CODE REVIEW

File: team.md [hunk 1]
```diff
@@ -13,4 +13,15 @@ permalink: /team/
      style="float: left; margin-right: 10px;" /> [Nicolás Quesada](https://www.polymtl.ca/expertises/en/quesada-nicolas).  
 Assistant Professor, [Engineering Physics](https://www.polymtl.ca/phys/), [Polytechnique Montréal](https://www.polymtl.ca).  
 MEI Chair in Quantum Photonics.  
-Enjoys riding his bike, taking pictures of birds and eating [alfajores](https://en.wikipedia.org/wiki/Alfajor).
\ No newline at end of file
+Enjoys riding his bike, taking pictures of birds and eating [alfajores](https://en.wikipedia.org/wiki/Alfajor).
+
+**Reasearch Group**
```
Comment (nquesada) [inline team.md:18]:
```suggestion
**Research Group**
```

File: team.md [hunk 2]
```diff
@@ -13,4 +13,15 @@ permalink: /team/
...
+
+<img src="https://clerkgroup.uchicago.edu/img/MartinWebPic.jpg"
+     alt="Martin"
+     width="80" 
+     height="90"
+     style="float: left; margin-right: 10px;" /> Martin Houde.  
+Postdoctoral Associate, [Engineering Physics](https://www.polymtl.ca/phys/), [Polytechnique Montréal](https://www.polymtl.ca).  
+Ph.D Mcgill.
```
[the remark this answers is not in the document]
Comment (nquesada) [reply team.md:26]: something like: Graduated from McGill University in 2018 with a Ph.D. thesis on [Parametric driving as a route to
topological states and quantum devices](https://escholarship.mcgill.ca/downloads/rj430699k) supervised by [A. Clerk](https://clerkgroup.uchicago.edu).

**Comment by nquesada** [comment]
Congrats @MHoude2 seems like your first PR is close to being ready. Please do not merge until you have addressed my comments.

File: team.md (hunk 1 above)
Comment (MHoude2) [reply team.md:18]: fixed typo

File: team.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MHoude2) [reply team.md:26]: modified

**Pull request merged**